app/assets/stylesheets/_cookie-banner.scss

```diff
@@ -0,0 +1,56 @@
+.cookies-banner {
+  background-color: $lightgrey-10;
+  position: fixed;
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;  
+  border-radius: 4px;
+  margin: 0 auto;
+  padding: 10px;
+  left: 20px;
+  right: 20px;
+  max-width: fit-content;
+  bottom: -400px;
+  box-shadow: $lightgrey-16 0px 0px 12px 0px;
+  z-index: 1000;
+  visibility: hidden;
+  transition: all 0.8s ease-in-out;  
+
+  .cookies-banner__msg {
+    margin: 0 20px 0 0;
+    text-align: justify;
+  }
+
+  .cookies-banner__btn {
+    background-color: $blue;
+    color: $lightgrey-20;
+    border: none;
+    border-radius: 4px;
+    padding: 8px 12px;
+    font-weight: bold;
+    text-transform: uppercase;
+
+    &:hover {
+      background-color: $blue-2;
+    }
+  }
+}
+
+@media (orientation: portrait) {
+  .cookies-banner {
+    flex-direction: column;
+
+    .cookies-banner__msg {
+      margin: 0 0 10px 0;
+    }
+
+    .cookies-banner__btn {
+      width: 100%;
+    }
+  }
+}
+
+.visible-banner {
```

Comment:
For this class, we could create a modifier class inspired by BEM standards, which could be called `cookies-banner--visible`, what do you think?